api/async/publish.ts

```diff
@@ -87,6 +87,9 @@ async function publishGithub(
   console.log(
     `Publishing ${build.options.moduleName} at ${build.options.ref} from GitHub`,
   );
+  const quota = await database.getOwnerQuota(
+    build?.options.repository.split("/")[0] as string,
```

Comment:
Why is the optional chaining necessary? It isn't 2 lines above in the `console.log`.

Reply:
autocomplete shenanigans 🤷‍♂️